libs/hdf-converters/src/twistlock-mapper.ts

```diff
@@ -21,6 +21,23 @@ const IMPACT_MAPPING: Map<string, number> = new Map([
   ['low', 0.3]
 ]);
 
+export class TwistlockResults {
+  data: Record<string, unknown>;
+  customMapping?: MappedTransform<ExecJSON.Execution, ILookupPath>;
```

Comment:
if you don't use this attribute, then don't have it here.  wish we didn't put in the custom mapping stuff in the first place cause i've only seen it used once or twice across all of our mappers.